### ISSUE TITLE
Translation work still to be merged into master branch

### ISSUE THREAD

**Issue by jc2820** [cybersafe-dev/cybersafety-tool#317]
Translation work has been completed except for the new PDF templates (in both English and Irish). Currently waiting on those before the full translation is pushed live (also see #314). Can be viewed on the lang-toggle branch.

Lang-toggle may need a merge from master before pull request - Lots of files changed in both master and lang-toggle. 

This issue mainly to remind that this is a job to be done.

**Comment by jc2820** [comment]
Merged with master 4/3/22.
Will likely need merging again before live, but the bulk of new additions to master are merged with lang-toggle now including node module updates.
